chall03/mtuomine.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const baseURL = "https://chall03.hive.fi/"
+
+func main() {
+	client := http.Client{Timeout: time.Second * 1}
+
+	// First request
+	res, err := client.Get(baseURL)
+	checkErr(err)
+
+	body, err := ioutil.ReadAll(res.Body)
+	checkErr(err)
+	res.Body.Close()
+
+	// Second request
+	res, err = client.Get(baseURL + createQuery(string(body)))
+	checkErr(err)
+	defer res.Body.Close()
+	fmt.Println(res.StatusCode)
+}
+
+func checkErr(err error) {
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func parseNumbers(str string) []string {
+	re := regexp.MustCompile("[0-9]+")
```

Comment:
You should avoid regex for such a project, here a nice post you could read: https://softwareengineering.stackexchange.com/a/113243